### PR TITLE
State route index with object

### PR DESCRIPTION
**Pull request by EdDev**
Depends on #388

### CODE REVIEW

File: libnmstate/nm/__init__.py [hunk 1]
```diff
@@ -36,6 +37,7 @@
 checkpoint
 connection
 device
+dns
```
Comment (cathay4t) [inline libnmstate/nm/__init__.py]: This change seems irrelevant to this PR. Can we remove it?

Comment (EdDev) [reply libnmstate/nm/__init__.py]: metadata module needs it as it uses nm.dns.
I do not know how it worked without it. It is not related to my changes.. strange.
